bettercli/cl/__init__.py

```
@@ -0,0 +1,4 @@
+from .colours import ANSIColors
```

Ruff (0.8.2)

`.colours.ANSIColors` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)